src/app/pages/categories/management/categories-editor/pages-categories-editor.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="loading-container" *ngIf="transactionTypeDisabledLoading">
+  <mat-spinner [diameter]="30"></mat-spinner>
+</div>
+
+<form [formGroup]="form" class="form__container" *ngIf="!transactionTypeDisabledLoading">
+  <h1 mat-dialog-title>{{ data.id ? "Update Category" : "Create Category" }}</h1>
+  <mat-form-field>
+    <mat-label>Select type</mat-label>
+    <mat-select [formControl]="form.controls['type']">
+      <mat-option *ngFor="let type of selectTransactionsTypes | keyvalue" [value]="type.value">
+        {{ type.key }}
+      </mat-option>
+    </mat-select>
+    <mat-hint *ngIf="typeAlreadyUsed">
+      Category is already used by transaction(s)
```

Comment:
This one should have bottom margin - is now way too close to the input below:
![Track Expenses 2023-03-04 19-39-51](https://user-images.githubusercontent.com/16034577/222923233-74c926da-4094-4f0c-a32b-05b4c97c518d.jpg)